lib/matplotlib/animation.py

```diff
@@ -1371,6 +1374,26 @@ def _repr_html_(self):
         elif fmt == 'jshtml':
             return self.to_jshtml()
 
+    def pause(self):
+        """Pause the animation."""
+        if not self._paused:
+            self._paused = True
+            self.event_source.stop()
+            self._fig.canvas.draw_idle()
+            if self._blit:
+                for artist in self._drawn_artists:
+                    artist.set_animated(False)
+            self._fig.set_animated(False)
```

Comment:
Didn't even realize `Figure` had a `set_animated`. Probably should call `set_animated(True)` in `resume()` for symmetery.

Reply:
`set_animated` comes from the `Artist` baseclass.  Given that we use `get_animated` to exclude things from the draw tree, I'm not sure it makes good sematic sense set / unset it on the Figure.

Reply:
Is there any change that needs to be happen here? Is the consensus it is good the way it is now?

Reply:
Well, either we need to call both `set_animated(False)` here in `pause()` and `set_animated(True)` in `resume()` or neither. Based on @tacaswell 's comment, I'd go with neither.

Reply:
Removed this in the latest commit.